### PR TITLE
Add Link for accession in batch details page

### PR DESCRIPTION
**Pull request by nickgraz**
- [x] Waiting on https://github.com/terraware/terraware-server/pull/1567

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #1895** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1895?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/1895?utm_source=stack-comment).

File: src/components/InventoryV2/BatchSummary.tsx [hunk 1]
```diff
@@ -48,7 +50,15 @@ export default function BatchSummary(props: BatchSummaryProps): JSX.Element {
         <OverviewItemCard isEditable={false} title={strings.TOTAL_WITHDRAWN} contents={batch.totalWithdrawn} />
       </Grid>
       <Grid item xs={2}>
-        <OverviewItemCard isEditable={false} title={strings.ACCESSION_ID} contents={batch.accessionId || ''} />
+        <OverviewItemCard
+          isEditable={false}
+          title={strings.ACCESSION_ID}
+          contents={
```
Comment (karthikbtf) [inline src/components/InventoryV2/BatchSummary.tsx:56]: use null contents if there's no accession info?

**Review by karthikbtf — APPROVED** [review]
Simple enough!!

**Review by karthikbtf — APPROVED** [review]
Thanks!!

**Pull request merged**